src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1840,7 +1840,7 @@ namespace winrt::TerminalApp::implementation
                 return;
             }
 
-            TermControl newControl{ controlSettings, controlConnection };
+            TermControl newControl{ *(winrt::get_self<TerminalSettings>(controlSettings)->CreateChild()), controlConnection };
```

Comment:
Oh gosh. Can you file a Code Health task to clean up all the different places in which we create a TerminalControl with its settings? I don't love that we had to make this fix in multiple places.